packages/keyring/src/pair/toJson.spec.ts

```diff
@@ -37,4 +38,58 @@ describe('toJson', (): void => {
       }
     });
   });
+
+  const ethereumPair = new Keyring({ type: 'ethereum' });
+  const pairOpts = {
+    meta: {
+      genesisHash: '0xc8d32cd0020b9da3804ef726f876604b7fcf5a799e687221a959f2dba5c78f18',
```

Comment:
Not sure that brings anything, this test is based on a real life example, so kept it as close to the reality.